### DISCUSSION TITLE
Announcing a parquet reader and writer for tablesaw

### DISCUSSION THREAD

**Discussion by ccleva** [jtablesaw/tablesaw#965] [Show and tell]
Hi all,

Following the discussion in issue #221, we just publicly released the first version of a parquet reader and writer for tablesaw.

It is available in the maven central repository, and all details are in our github repository there:
https://github.com/tlabs-data/tablesaw-parquet

As this is our first release, please give us some feedback and let us know of any issue you might have with the library.

Enjoy!

@lwhite1  how should we reference this external reader in the tablesaw documentation ? Should I submit a PR with a new section for external supporting projects in the README ?

**Comment by lwhite1** [comment]
@ccleva I think a README PR is the best approach.  Thanks again. Looks like you did a great job on the documentation as well.

**Reply by ccleva** [reply]
Thanks, we tried ! I made PR #966 to add a link to the project page in the README.

Feel free to modify the text yourself or ask me to do it.

**Comment by ccleva** [comment]
Hi all,
A quick message to announce our new release (v0.10.0) with support for reading remote files.
Enjoy!

**Reply by lwhite1** [reply]
That's awesome. Thank you for letting us know.

**Reply by lwhite1** [reply]
Hi @ccleva. I don't know if you've seen it, but there's a nice write-up on your work here: https://medium.com/@gmsharpe/io-parquet-with-java-on-colab-27806dbedfd9

You should post this to the parquet-user mailing list as well if you haven't already.

**Reply by ccleva** [reply]
Thank you @lwhite1, I was not aware of that article !

I have not yet shared our work with the parquet community, you are right I should.
